### DISCUSSION TITLE
Drawer: Expose appendTo property of Portal

### DISCUSSION THREAD

**Discussion by gigonzalez-shipbob** [primefaces/community#3234] [PrimeVue]
Hello!

We use primevue inside of shadow DOM. Without access to appendTo prop of Portal it is difficult to override the work Teleport does to append it to the default body element.

Proposed PR: https://github.com/primefaces/primevue/pull/6971

**Comment by DigitalCommotion** [comment]
Any reason why this was closed?

**Comment by timpulver** [comment]
+1

For a current project I would need an `appendTo` prop of the Drawer component in order to use the Drawer inside a fullscreen element (via Fullscreen API).
When the Drawer is rendered outside of the DOM element in fullscreen mode, it remains hidden.
